src/core_modules/capture-core/components/WorkingLists/TeiWorkingLists/Setup/hooks/useStaticTemplates.js

```diff
@@ -3,11 +3,11 @@ import { useMemo } from 'react';
 import i18n from '@dhis2/d2-i18n';
 import type { WorkingListTemplate } from '../../../WorkingListsBase';
 
-export const useStaticTemplates = (defaultAlteredTemplate?: WorkingListTemplate) =>
+export const useStaticTemplates = (defaultAlteredTemplate?: WorkingListTemplate, defaultTemplateId: string) =>
     useMemo(
         () => [
             defaultAlteredTemplate || {
-                id: 'default',
+                id: defaultTemplateId ?? 'default',
```

Comment:
I think you can safely remove the `'default'` fallback string and just use `id: defaultTemplateId`

Reply:
Ah, thanks!